install.yaml

```
@@ -85,7 +86,7 @@ ddev_version_constraint: '>= v1.24.2'
 
 # List of add-on names that this add-on depends on
 dependencies:
-  # - redis
+   # - redis 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)